Common/Charting.cs

```diff
@@ -135,6 +139,17 @@ public class Series
         /// </summary>
         public SeriesType SeriesType = SeriesType.Line;
 
+        /// <summary>
+        /// Color the series 
+        /// </summary>
+        [JsonConverter(typeof(ColorJsonConverter))]
+        public Color Color = Color.Black;
```

Comment:
Agreed - null can be an indication for consumers of the chart data to autogen a color